dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/tai/TAISubjectUtils.java

```diff
@@ -183,18 +184,92 @@ void setUserNameFromJwtClaims(SocialLoginConfig config) {
     String getRealm(AttributeToSubject attributeToSubject, SocialLoginConfig config) throws SettingCustomPropertiesException {
         String realm = attributeToSubject.getMappedRealm();
         if (realm == null) {
+            realm = getDefaultRealm(config);
+            //realm = getDefaultRealmFromAuthorizationEndpoint(config);
```

Comment:
Can just delete this line rather than comment it out.

Reply:
done